File: Array/2D Array/2DArray.java
import java.util.Scanner;
public class TwoDarray
{
  public static void main(String args[])
  {
    int[][] arr = new int[3][3];
    Scanner sc = new Scanner(System.in);
    for(int i=0;i<3;i++)
    {
     for(int j=0;j<3;j++)
     {
      System.out.print("Enter elements");
      arr[i][j] = sc.next();
      System.out.println();
     }
    }
    System.out.println("Printing elements...");
    for(int i=0;i<3;i++)
    {
     System.out.println();
     for(int j=0;j<3;j++)
     {
      System.out.print(arr[i][j]+"\t");
     }
    }
   }
  }

File: Array/2D Array/2DArray.cs
***** C# program *****
using System;
public class Program
{
 public static void Main()
 {
  int[] arr = new int[3,3];
  for(int i=0;i<3;i++)
  {
   for(int j=0;j<3;j++)
   {
    Console.WriteLine("Enter element");
    arr[i,j]=Convert.Tolnt32(Console.ReadLine());
   }
  }
  Console.WriteLine("Printing Elements...");
  for(int i=0;i<3;i++)
  {
   Console.WriteLine();
   for(int j=0;j<3;j++)
   {
    Console.Write(arr[i,j]+"");
   }
  }
 }
} 
